src/__test__/Detail.test.js

```diff
@@ -1,21 +1,21 @@
-import React from "react";
-import { render, screen } from "@testing-library/react";
-import { Provider } from "react-redux";
-import thunk from "redux-thunk";
-import configureStore from "redux-mock-store";
-import { BrowserRouter } from "react-router-dom";
-import Detail from "../pages/Detail";
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import configureStore from 'redux-mock-store';
+import { BrowserRouter } from 'react-router-dom';
+import Detail from '../pages/Detail';
 
 const mockStore = configureStore([thunk]);
 
-describe("Detail Component", () => {
-  it("renders game details properly", () => {
+describe('Detail Component', () => {
+  it('renders game details properly', () => {
```

Comment:
- [ ] Please kindly take note that five out of the written tests are not passing. Carefully pay attention to the error message in order to resolve it appropriately. 👍 

<img width="296" alt="Untitled" src="https://github.com/naolkidanu/game-web-app/assets/103386183/283a8906-3f57-48cc-a8d9-0e55204916b2">